config.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 the generic-device-plugin authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"github.com/squat/generic-device-plugin/deviceplugin"
+	"gopkg.in/yaml.v3"
```

Comment:
I would rather use

Reply:
Oh cool! I'll do that. `gopkg.io/yaml` v2 or v3 seems to be more standard and is the one I've always used, but I like being able to leave the tags as is.

Reply:
🙇